packages/dmn-js-decision-table/src/features/decision-rules/components/DecisionRulesCellEditorComponent.js

```diff
@@ -167,8 +163,12 @@ class TableCellEditor extends Component {
       businessObject.expressionLanguage !== defaultExpressionLanguage;
   }
 
-  getDefaultExpressionLanguage(businessObject) {
-    const elementType = is(businessObject, 'dmn:UnaryTests') ? 'inputCell' : 'outputCell';
+  _isInputCell() {
+    return is(this.props.businessObject, 'dmn:UnaryTests');
+  }
+
+  getDefaultExpressionLanguage() {
+    const elementType = this._isInputCell() ? 'inputCell' : 'outputCell';
```

Comment:
As I understand this, This is just a refactoring to get the businessObject from the props instead of having it passed to the function, correct?

If so, we can then also adjust the function call in `isScript` for consistency. (non-blocking, can be done as follow-up)

https://github.com/bpmn-io/dmn-js/blob/49a586d6a6e5abf548b65afc14bde2163ab998ae/packages/dmn-js-decision-table/src/features/decision-rules/components/DecisionRulesCellEditorComponent.js#L152

Reply:
OK